scripts/publish/combine_minify_css.R

```diff
@@ -0,0 +1,44 @@
+publish.combine_minify_css <- function(viz) {
```

Comment:
sorry. getting picky about names. But if this doesn't minify, it should be `combine_css`

Reply:
true, I did leave the code that we would use to minify in here commented out in case we want to reference it in the future. That's why I didn't change the name. But I see your point - let me know if you still think that's a good idea

Reply:
I'll defer to you. I don't want to hold up a PR because of a name change.

Reply:
easy fix, so I'll change it